pkg/ddevapp/compose_yaml.go

```diff
@@ -56,6 +58,7 @@ func (app *DdevApp) WriteDockerComposeYAML() error {
 	}
 	fullContents, _, err := dockerutil.ComposeCmd(&dockerutil.ComposeCmdOpts{
 		ComposeFiles: files,
+		Profiles:     []string{nodeps.ComposeProfileOptional},
```

Comment:
> I haven't figured out how to support more than one profile, mostly because the `docker-compose config` command only brings in all the profiles if you specify them.

Does this solve the problem described?

```suggestion
		Profiles:     []string{"*"},
```

The docs say:

If you want to enable all profiles at the same time, you can run `docker compose --profile "*"`

https://docs.docker.com/compose/how-tos/profiles/#start-specific-profiles

Reply:
However, is there a difference between that and "optional" in the end? What is clearer for our users?

I'll definitely consider "*".

Reply:
Oh, this may solve things for me though! I cn use "*" in the `docker-compose config`, and then I'll be able to use arbitrary profiles in the start. Thanks!

Reply:
We can use `--optional` as quick and easy way for `optional` profile and add a new flag that gives people more flexibility, i.e. use the same `--profile` (array flag) as in `docker-compose` for a broader approach if it makes sense. Flags must be mutually exclusive.